### ISSUE TITLE
More Documentation on Rules for Identifying Parquet Files

### ISSUE THREAD

**Issue by dbready** [duckdb/duckdb-web#130]
Perhaps this is nothing more than my mis-reading of the website documentation, but I have a collection of parquet files that are saved with a '.parq' extension. It was only after significantly banging my head against the wall that I realized that Duck would not identify the extension as valid and was failing to interrogate the files.

```python
con.execute(
"""
select *
from '/abs_path/2018_01_02.parq'
limit 10
""").fetch_df()
```
Returns ```RuntimeError: Catalog Error: Table with name /abs_path/2018_01_02.parq does not exist!
LINE 3: from '/abs_path/2018_01_02.parq'```

If I rename a file to .parquet extension, everything works as intended. This failure happened with both specific filenames and the glob syntax.

Not sure how common .parq is in the wild and if you need to support this, but it would be nice if the full rules for identifying queryable file extensions was more clear.

**Comment by Alex-Monahan** [comment]
Thank you for the recommendation! It looks like the documentation does mention it somewhat,  but we could call more attention to it:
"If your file ends in .parquet, the parquet_scan syntax is optional. "

One alternative that should work without changing the file extension is the parquet_scan table function. Ex:
```sql
Select * from parquet_scan('filename.parq')
```

**Comment by dbready** [comment]
Yes, the documentation example with parquet_scan was there, but I failed to appreciate the significance as everything on the page ended in .parquet. Maybe if the parquet_scan function was moved to the top of the page before the examples block? To make it more explicit that it is getting used under the hood for the following snippets?

Definitely my own misunderstanding, but appreciate anything that might make it a bit more clear for anyone else that stumbles into the same situation.

**Comment by Mytherin** [comment]
I have added this scenario [to the examples on the top of the page](https://duckdb.org/docs/data/parquet):

```sql
-- if the file does not end in ".parquet", use the parquet_scan function
SELECT * FROM parquet_scan('test.parq');
```

**Comment by Alex-Monahan** [comment]
Thanks for adding that example @Mytherin, I think that solves this!
